lib/main_common.pm

```diff
@@ -480,7 +480,15 @@ sub load_extra_tests {
             }
 
         }
-        loadtest 'x11/yast2_lan_restart' if check_var('DISTRI', 'gnome');
+        # the following tests care about network and need some DE specific
+        # needles. For now we only have them for gnome and do not want to
+        # support more than just this DE. Probably for later at least the wifi
+        # test, checking the wifi applet, would make sense in other DEs as
+        # well
+        if (check_var('DESKTOP', 'gnome')) {
+            loadtest 'x11/yast2_lan_restart';
```

Comment:
I was talking about our variable name, apart from it's value. And second point is still valid, if we have some extra checks. In the end it's matter of taste, but I find it bad to have hardcoded strings and "magic" numbers all over the place. If it would be simple condition check, it doesn't worth it to put it to separate function.